tests/Helper/WorkQueueTest.php

```diff
@@ -160,4 +160,113 @@ public function testPatientStatusGroupExportSelectedHeaders()
             'Core Participant Minus PM Date'
         ], $exportHeaders);
     }
+
+    public function testGetFilterLabelOptionPairs()
+    {
+        $filterLabelOptionPairs = [
+            'labels' => [
+                'enrollmentStatus' => 'Participant Status',
+                'activityStatus' => 'Activity Status',
+                'patientStatus' => 'Patient Status',
+                'consentForStudyEnrollment' => 'Primary Consent',
+                'questionnaireOnDnaProgram' => 'Program Update',
+                'consentForElectronicHealthRecords' => 'EHR Consent Status',
+                'consentForGenomicsROR' => 'gRoR Consent Status',
+                'consentForDvElectronicHealthRecordsSharing' => 'DV-Only EHR Sharing',
+                'consentForCABoR' => 'CABoR Consent',
+                'consentCohort' => 'Consent Cohort',
+                'primaryLanguage' => 'Language of Primary Consent',
+                'ageRange' => 'Age',
+                'race' => 'Race',
+                'genderIdentity' => 'Gender Identity',
+                'isEhrDataAvailable' => 'EHR Data Transfer',
+                'ehrConsentExpireStatus' => 'EHR Expiration Status',
+                'retentionType' => 'Retention Status',
+                'retentionEligibleStatus' => 'Retention Eligible',
+                'participantOrigin' => 'Participant Origination',
+                'enrollmentSite' => 'Enrollment Site',
+                'consentForStudyEnrollmentAuthoredStartDate' => 'Primary Consent Start Date',
+                'consentForStudyEnrollmentAuthoredEndDate' => 'Primary Consent End Date',
+                'questionnaireOnDnaProgramAuthoredStartDate' => 'Program Update Start Date',
+                'questionnaireOnDnaProgramAuthoredEndDate' => 'Program Update End Date',
+                'consentForElectronicHealthRecordsAuthoredStartDate' => 'EHR Consent Status Start Date',
+                'consentForElectronicHealthRecordsAuthoredEndDate' => 'EHR Consent Status End Date',
+                'consentForGenomicsRORAuthoredStartDate' => 'gRoR Consent Status Start Date',
+                'consentForGenomicsRORAuthoredEndDate' => 'gRoR Consent Status End Date',
+                'consentForDvElectronicHealthRecordsSharingAuthoredStartDate' => 'DV-Only EHR Sharing Start Date',
+                'consentForDvElectronicHealthRecordsSharingAuthoredEndDate' => 'DV-Only EHR Sharing End Date',
+                'consentForCABoRAuthoredStartDate' => 'CABoR Consent Start Date',
+                'consentForCABoRAuthoredEndDate' => 'CABoR Consent End Date',
+                'ehrConsentExpireStatusAuthoredStartDate' => 'EHR Expiration Status Start Date',
+                'ehrConsentExpireStatusAuthoredEndDate' => 'EHR Expiration Status End Date',
```

Comment:
Instead of copy / pasting this data, why don't we just call the static `WorkQueue::filterDateFieldLabels`?